docs/src/components/DocsMDXProvider.js

```diff
@@ -29,7 +29,21 @@ Components.strong = props => <strong className="strong" {...props} />
 
 export default class DocsMDXProvider extends React.PureComponent {
   static propTypes = {
-    children: PropTypes.node
+    children: PropTypes.node,
+    noInline: PropTypes.bool
+  }
+
+  static defaultProps = {
+    noInline: false
+  }
+
+  constructor(props) {
+    super(props)
+
+    // This feels hacky...
+    Components.code = newProps => (
```

Comment:
Can you document the purpose of it?

Reply:
I needed to pass the `noInline` prop to the `react-live` component in order to show full examples in the theme docs rather than just a single component.

Do you know of a better way to do this? or should adding a comment here suffice

Reply:
Curious if we could instead do this on a component-by-component basis? That would avoid needing to propagate it from `DocsMDXProvider` to the `.code` component right?

Reply:
How would it work on a component-by-component basis?

I'm thinking alternatively we could just have the `ThemeLayout` have its own MDXProvider  since I don't think there'll be another case where the `noInline` prop will be needed for `react-live`